svelte-kit-scss/src/routes/(authenticated)/[username]/[repo]/[issueSearchType=issue_search_type]/+page.server.ts

```diff
@@ -0,0 +1,58 @@
+import type { PageServerLoad } from './$types';
+import {
+  IssueSearchQueryState,
+  IssuesSearchQuerySort,
+  type IssueSearchQueryFilter,
+} from '$lib/constants/issues-search-query-filters';
+import {
+  buildIssueSearchQuery,
+  ensureIssuesSearchQueryFilterValue,
+  parseIssuesSearchQueryAllowed,
+} from '$lib/helpers/issues-search-query-builder';
+import { IssuesSearchService } from '$lib/services';
+import { IssueSearchPageTypeFiltersMap, type IssueSearchTypePage } from '$lib/constants/matchers';
+
+const DEFAULT_PER_PAGE = 25;
```

Comment:
Honest question: Why put the default here instead of in the service class?
_(not requesting a change just curious)_

Reply:
Right, taking a param/argument seems okay, but why have the default `25` be here in the route?